domovoi.py

```diff
@@ -13,6 +13,19 @@
 
 
 class Domovoi:
+
+    '''
+    Attempt to Ping the other pi, if the pi responds open primary processes, otherwise open secondary processes
+    '''
+    def ping_raspi(self):
```

Comment:
This does more than just ping the raspi pi, it also picks the settings file to pick. This function name is in aptly named at the very least. Also it doesn't make sense for Domovoi to have a function on how to initialize itself, this should go outside of the Domovoi class.